cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -101,9 +102,37 @@ public void run() {
         @CommandLine.Mixin
         LoginCommandOptions loginOptions;
 
+        @CommandLine.Mixin
+        CreateControllerCommandOptions commandOptions;
+
         @Override
         public void run() {
-            // TODO:Implement functionality
+
+            String yamlFileContent;
+            try {
+                yamlFileContent = FileUtils.readLocalFile(commandOptions.getYamlFilePath());
+            } catch (IOException e) {
+                System.err.println(e.getMessage());
+                return;
+            }
+            Yaml yamlLoader = YamlCreator.createDefaultYamlProcessor();
+            ApplicationBean applicationBean = yamlLoader.loadAs(yamlFileContent, ApplicationBean.class);
+            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
+
+            ApplicationOperations applicationOperations = new ApplicationOperations(cfOperations);
+            try {
+                applicationOperations.create(applicationBean, true);
+                System.out.println("App created");
```

Comment:
Would be nice to print the name of the created application too.

Reply:
done